exporter/opentelemetry-exporter-datadog/src/opentelemetry/exporter/datadog/exporter.py

```diff
@@ -312,19 +313,23 @@ def _parse_tags_str(tags_str):
     return parsed_tags
 
 
-def _extract_tags_from_resource(resource):
+def _extract_tags_from_resource(resource, fallback_service_name):
     """Parse tags from resource.attributes, except service.name which
     has special significance within datadog"""
     tags = {}
     service_name = None
```

Comment:
Since it's not used below on line 321 anymore, I think it would make sense to push the initialization of `service_name = None` lower to line 322 perhaps?

Reply:
makes sense, updated!